storage/adls_gen1_mount.go

```diff
@@ -24,13 +26,20 @@ func (m AzureADLSGen1Mount) Source() string {
 }
 
 // Config ...
-func (m AzureADLSGen1Mount) Config() map[string]string {
+func (m AzureADLSGen1Mount) Config(client *common.DatabricksClient) map[string]string {
+	azureEnvironment, err := client.AzureAuth.GetAzureEnvironment()
+	var clientEndpoint string
+	if err != nil {
+		clientEndpoint = azure.PublicCloud.ActiveDirectoryEndpoint
```

Comment:
hm... `Config` function has no error returned. and you're not logging this error anywhere.

Ideally, you can add `AzureEnvironment` field to `AzureAuth`, which you'll set from `configureWithClientSecret` and `configureWithAzureCLI` calling the `getAzureEnvironment`. this way you'll have guarantees to propagate errors early on and have no side-effects for this function.